src/main/CMakeLists.txt

```diff
@@ -75,7 +75,7 @@ elseif(OS_IS_MAC)
     set(MACOSX_BUNDLE_LONG_VERSION_STRING ${MUSESCORE_VERSION_FULL})
     set(MACOSX_BUNDLE_SHORT_VERSION_STRING ${MUSESCORE_VERSION})
     set(MACOSX_BUNDLE_BUNDLE_VERSION ${MUSESCORE_VERSION_FULL})
-    set(MACOSX_BUNDLE_COPYRIGHT "Copyright © 1999-2021 MuseScore BVBA and others. Published under the GNU General Public License.")
+    set(MACOSX_BUNDLE_COPYRIGHT "Copyright © 1999-2022 MuseScore BVBA and others. Published under the GNU General Public License.")
```

Comment:
Shouldn't a " version 3" get added here?

Reply:
Yes, good idea (I see you added it already), so that it's at least consistent with the other places.